src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -203,11 +203,6 @@ public void onFormStatusEvent(FormStatusEvent event) {
     updateForms();
   }
 
-  @EventSubscriber(eventClass = ExportProgressEvent.class)
```

Comment:
This removal and the next addition looks like you just reordered the methods on the file. Let's revert this change to keep changes to a minimum, plz :)

Reply:
Ok will fix that. I actually deleted the method and had a new one that was listening to ExportProgressPercentageEvent with name onExportProgressPercentageEvent but since you said the progress event is only useful for a single export I deleted the method and added this one back. Didn't realize I'd change the positions. Will undo it.